R/tax_certainty.R

```diff
@@ -0,0 +1,33 @@
+#' Check if a taxon is certain or uncertain.
+#'
+#' Check whether a given taxon in a vector is certain or uncertain.
+#' Binomial names (genus and species) are considered "certain",
+#' whereas names containing only the genus,
+#' or genus and not determined species is considered "uncertain".
+#'
+#' @param taxon_names Character vector containing the species names to be
+#' checked.
+#' It allows taxon names separated by underscore or space.
+#' @return Numeric vector classifying each species as certain (1) or uncertain
```

Comment:
I think @bethany-j-allen envisioned this function as such to feed into `tax_range_strat` but I am wondering if as a standalone function it makes more sense to be logical (TRUE/FALSE)

Reply:
I don't think it is confusing if you frame it in the docs as "the taxonomic identification is certain", TRUE or FALSE. That said, I like your second option. It could be envisioned as something like:
- `tax_certainty(certain = TRUE, uncertain = FALSE)` or 
- `tax_certainty(def = list(certain = 1, uncertain = 0)`